hal/armv7m/imxrt/117x/imxrt.c

```diff
@@ -332,8 +340,7 @@ u8 _imxrt_nvicGetPriority(s8 irqn)
 void _imxrt_nvicSystemReset(void)
 {
 	*(imxrt_common.scb + scb_aircr) = ((0x5fa << 16) | (*(imxrt_common.scb + scb_aircr) & (0x700)) | (1 << 0x02));
-
-	__asm__ volatile ("dsb");
```

Comment:
The same `__asm__` code is in:
https://github.com/phoenix-rtos/plo/blob/acd42a0b7c03c3c17fc7d90fb9db3831e1af3fde/hal/armv7m/imxrt/10xx/imxrt.c#L1553

Can you change it there as well?